code/datums/elements/ash_storm_listener.dm

```diff
@@ -0,0 +1,37 @@
+///This element just handles creating and destroying an area sound manager that's hooked into ash storm stuff
+/datum/element/ash_storm_listener
```

Comment:
this element is too specific, especially if given to all clients on mob/Login(). if someone wanted to add a new weather type theyd need to create another almost identical element just for that.

Reply:
yeah, sure. I'll make it bespoke, and change it to be based on weather type